Tests/test_file_ppm.py

```diff
@@ -13,16 +13,64 @@
 
 def test_sanity():
     with Image.open(TEST_FILE) as im:
-        im.load()
         assert im.mode == "RGB"
         assert im.size == (128, 128)
-        assert im.format, "PPM"
+        assert im.format == "PPM"
         assert im.get_format_mimetype() == "image/x-portable-pixmap"
 
 
+def test_arbitrary_maxval():
```

Comment:
Can these mode be parametrised? Looks like one input bytestring and two things to assert.

Reply:
Ok, done

Reply:
Thanks!